tests/functional/algos/test_algos.py

```diff
@@ -383,6 +391,41 @@ def test_with_fidelity(algorithm: dict):
     assert param.type == "real"
 
 
+@pytest.mark.parametrize(
+    "algorithm",
+    algorithm_configs.values(),
+    ids=list(algorithm_configs.keys()),
+)
+def test_seeding(algorithm: dict):
+    """Test seeding works properly."""
+
+    rosenbrock_with_fidelity.initialize()
+    exp = workon(
+        rosenbrock_with_fidelity,
+        space_with_fidelity,
+        algorithms=algorithm,
```

Comment:
Is the PR for renaming "algorithms" -> "algorithm" in yet?
```suggestion
        algorithm=algorithm,
```

Reply:
Not yet but good catch. Let's harmonize already. Thanks!